packages/paste-website/src/pages/components/editable-code-block/index.mdx

```diff
@@ -0,0 +1,370 @@
+export const meta = {
+  title: 'Editable CodeBlock',
+  package: '@twilio-paste/editable-code-block',
+  description: 'An Editable Code Block is a text field that allows users to enter formatted code.',
+  slug: '/components/editable-code-block/',
+};
+
+import Changelog from '@twilio-paste/editable-code-block/CHANGELOG.md';
+import packageJson from '@twilio-paste/editable-code-block/package.json';
+
+import {DoDont, Do, Dont} from '../../../components/DoDont';
+import {Blockquote} from '../../../components/Blockquote';
+import {Codeblock} from '../../../components/codeblock';
+import {SidebarCategoryRoutes} from '../../../constants';
+import DefaultLayout from '../../../layouts/DefaultLayout';
+import {getFeature, getNavigationData} from '../../../utils/api';
+
+export default DefaultLayout;
+
+export const getStaticProps = async () => {
+  const navigationData = await getNavigationData();
+  const feature = await getFeature('Editable Code Block');
+  return {
+    props: {
+      data: {
+        ...packageJson,
+        ...feature,
+      },
+      navigationData,
+    },
+  };
+};
+
+<NormalizedComponentHeader
+  categoryRoute={SidebarCategoryRoutes.COMPONENTS}
+  storybookUrl="/?path=/story/components-editable-code-block--"
+  githubUrl="https://github.com/twilio-labs/paste/tree/main/packages/paste-core/components/editable-code-block"
+  data={props.data}
+/>
+
+---
+
+<contentwrapper>
+
+<PageAside data={mdxHeadings} />
+
+<content>
+
+<StoryPreview
```

Comment:
non-blocking: there's a ton of whitespace around the codeblock in these previews, is there anything we can do about that?

Reply:
Can't figure it out easily. Removing the padding only covers top left and right, but the border edges still peek through and there main issue is bottom whitespace due to not being able to determine height correctly
<img width="776" alt="image" src="https://github.com/twilio-labs/paste/assets/1592327/bb5fc0d1-05f2-4ca2-a66c-1476c031cf15">
Just going to leave it as-is for now